blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java

```diff
@@ -234,7 +234,19 @@ public HttpResponse submitInputStep(StaplerRequest request) {
             }
 
             //XXX: execution.doProceed(request) expects submitted form, otherwise we could have simply used it
-            execution.preSubmissionCheck();
+
+            try {
+                execution.preSubmissionCheck();
+            } catch (Failure f) {
+                return new HttpResponse() {
```

Comment:
@NicuPascu Ah, Jacoco failure is legitimate. There is no test for this code block. Please look at https://github.com/jenkinsci/blueocean-plugin/blob/2c91571484e3524a1449719f9fdce059dbf1808d/blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeTest.java#L2115 and other input related tests and add one for this case.